static/templates/pod.html

```diff
@@ -0,0 +1,12 @@
+<div class="panel panel-default">
```

Comment:
@rowanseymour re your [templates question](https://github.com/rapidpro/casepro/pull/107#discussion_r71674335), we can do this. The tricky thing is getting haml templates working in karma tests for us to test our directive.

I've been trying out the karma haml preprocessors out there, none of them giving us what we need to be able to test:

  - [karma-ng-haml2js-preprocessor](https://github.com/codysims/karma-ng-haml2js-preprocessor): appears to look for a `haml` executable
  - [karma-haml-preprocessor](https://github.com/ondrasak/karma-haml-preprocessor): compiles to js strings that can be `eval`ed to raw html, but we're wanting to pipe its results to [karma-ng-html2js-preprocessor](https://github.com/karma-runner/karma-ng-html2js-preprocessor), which is expecting raw html

If we don't need to worry about using haml for this template and only need to keep the template in the `templates/partials/` directory, then this should be easy to change. Would this be alright?